web/themes/custom/pineapple/components/_comments.twig

```diff
@@ -83,10 +83,9 @@
     indicator here would break the render cache.
   #}
   <mark class="hidden new-label" data-comment-timestamp="{{ new_indicator_timestamp }}"></mark>
-
   {% set submitted_attributes = submitted_attributes|default(create_attribute()).addClass([
     'comment-submitted',
-    author_id == 1 ? 'admin-comment',
+    author_id == 1 or author_id == commented_entity.getOwnerId() ? 'highlighted-comment'
```

Comment:
Author id here, is this the user id of the author?

Reply:
author ID is the user ID of the person writing the comment

Reply:
Okay so why are you using "1" as a part of this check?

Reply:
Because that's the admin.

Reply:
Okay, why are you using the Id though, can you categorically guarantee you're only ever gonna have one admin on the site?

Reply:
I was going to say yes, but actually, it doesn't matter. If the code doesn't accommodate the future possibilities it's not good enough, so I'll update this.

Reply:
Done